### ISSUE TITLE
Documents history

### ISSUE THREAD

**Issue by lej333** [lej333/teambox#4]
Shows all documents which were sent or received to the customer. This documents will be saved in a Google Drive folder with name and e-mail address of the customer.

**Comment by Flare0** [comment]
Can choose to have a Team Drive dedicated for this or a specific folder in a Personal Drive.

**Comment by lej333** [comment]
Yes, and I think we have to ask for permission to write within as well. We have to investigate it. @Flare0
